src/main.rs

```diff
@@ -367,88 +265,36 @@ fn main() -> Result<(), String> {
                 }
             }
 
-            new_state = current_orient.new_state;
-            x_state = current_orient.x_state;
-            matrix = current_orient.matrix;
-
-            if new_state != old_state {
+            if current_orient.wayland_state != old_state {
+                let old_env = transform_to_env(&old_state);
+                let new_env = transform_to_env(&current_orient.wayland_state);
```

Comment:
I'm not really sure what the best way to set these env variables is now. Before #45 was merged, I switched the tracking variable to use the `Transform` enum from wayland-rs, since the values in `new_state` were no longer needed anywhere else. and it got rid of some bidirectional mapping code.

For now I've kept the user-facing API identical to #45, but imo it's a bit confusing because `swaymsg` and wayland transforms use opposite rotation directions for some reason. I think the best way to go would be setting the `ORIENTATION` and `PREV_ORIENTATION` env variables using the less-ambiguous `x_state` values. What do you think?